mlx/xunit2rst.mako

```diff
@@ -82,10 +85,13 @@ if not len(suite):
     continue
 
 for test in suite:
-    if len(test):
+    if test.findall('failure'):
         test_result = 'Fail'
         relationship = 'fails'
         break
+    if test.findall('skipped'):
+        test_result = 'Skip'
+        relationship = 'skipped'
```

Comment:
there is break in step above, but not here?

Reply:
That is intentional. If any test case in a suite has failed, the result 'Fail' is assigned. If a test case has been skipped, but a following test case in the same suite has failed, we still want to assign the result 'Fail'. This is what I came up with anyways. Do you agree?

Reply:
yes, I do. Because then skipped is exactly the same as fail, but in fact skipped is acceptable failure, so it should not mark the whole test suite as failure - otherwise we might as well not have skipped it.